Source/Icebreaker/Bot/IcebreakerBot.cs

```diff
@@ -34,6 +37,7 @@ public class IcebreakerBot : TeamsActivityHandler
         private readonly MicrosoftAppCredentials appCredentials;
         private readonly TelemetryClient telemetryClient;
         private readonly string botDisplayName;
+        private string teamsViewCardId;
```

Comment:
Okay found the answer to how this is working. It turns out the [bot is registered as a singleton](https://github.com/OfficeDev/microsoft-teams-apps-icebreaker/blob/8dd87c62606b94efbdfdb440365166809c492733/Source/Icebreaker/IcebreakerModule.cs#L28) so it re-uses the same instance for every request/activity.

Reply:
Every user of the bot will share this state...

Reply:
Changed this so that I'm now storing the activity ID of a user's most recent teams view card in the database and deleting the card whenever another message is received. This also avoids confusion with situations like clicking "pause all teams" and having the card not update or clicking "view my teams" multiple times and receiving multiple cards.